### PR TITLE
Show images for FIELD std_dev in the plotter

### LINKED ISSUE #7494: Visualise std of FIELDs (and SURFACES?)
After https://github.com/equinor/ert/commit/501709ce4a39f92ef0dc9973cbda81b11c625176 we now have an endpoint for calculating std for parameters, would be good to visualize this in the plotter. Something like a per layer std for FIELDS for example. First pass could for example be making images on the server and passing those through the API if that is easier.

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #7494 

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by frode-aarstad** [comment]
For the `test_field_param_update` test in `test_field_parameter.py` the following is generated: 

bilde

**Comment by dafeda** [comment]
image

**Comment by frode-aarstad** [comment]
image

File: src/ert/dark_storage/endpoints/records.py [hunk 1]
```diff
@@ -72,8 +71,9 @@ async def get_ensemble_record(
     ensemble_id: UUID,
     accept: Annotated[Union[str, None], Header()] = None,
 ) -> Any:
-    dataframe = data_for_key(storage.get_ensemble(ensemble_id), name)
+    import io
```
[the remark this answers is not in the document]
Comment (frode-aarstad) [reply src/ert/dark_storage/endpoints/records.py:74]: Ill move the imports to top of file as this seems to be the current recomendation

File: src/ert/dark_storage/endpoints/records.py [hunk 2]
```diff
@@ -138,3 +138,29 @@ def get_ensemble_responses(
...
+    import base64
+    import io
+
+    import matplotlib.pyplot as plt
+
+    ensemble = storage.get_ensemble(ensemble_id)
+    try:
+        da = ensemble.calculate_std_dev_for_parameter(key)["values"]
+    except:
+        return js.ImageOut(image=bytearray())
```
[the remark this answers is not in the document]
Comment (frode-aarstad) [reply src/ert/dark_storage/endpoints/records.py:158]: Added exception type for more info

File: src/ert/dark_storage/endpoints/records.py [hunk 3]
```diff
@@ -138,3 +138,29 @@ def get_ensemble_responses(
...
+
+    if z >= int(da.shape[2]):
```
[the remark this answers is not in the document]
Comment (frode-aarstad) [reply src/ert/dark_storage/endpoints/records.py:160]: Not sure I follow

File: src/ert/gui/plottery/plots/std_dev.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ert.gui.plottery import PlotContext
+
+
+class StdDevPlot:
+    def __init__(self):
+        self.dimensionality = 3
+
+    def plot(
```
[the remark this answers is not in the document]
Comment (frode-aarstad) [reply src/ert/gui/plottery/plots/std_dev.py:15]: It works fine with plots from different ensembles

File: src/ert/dark_storage/endpoints/records.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (oyvindeide) [reply src/ert/dark_storage/endpoints/records.py:156]: Was thinking like for example for [records](https://github.com/equinor/ert/blob/d30a2143ecaa00525fc45690eed7852be873d118/src/ert/dark_storage/endpoints/records.py#L73) and data type, but in this case I guess query parameter makes more sense?

```python
    accept: Annotated[Union[str, None], Header()] = None,
```

**Review by oyvindeide — APPROVED** [review]
LGTM! Just a minor comment, but not sure about it, so address it if you agree. Remember to squash before merging.

**Pull request merged**